dbfixture/fixture.go

```diff
@@ -233,6 +233,10 @@ func (f *Fixture) decodeField(strct reflect.Value, field *schema.Field, value ya
 		}
 	}
 
+	if v, ok := iface.(yaml.Unmarshaler); ok {
+		return v.UnmarshalYAML(value)
```

Comment:
the issue that's I am working on is still big number. yaml decoder, predicts the type for integers and treat the bigger ones as float. Inside UnmarshalYAML, one should even change the decoder prediction. 

```
var _ yaml.Unmarshaler = (*Bigint)(nil)

func (b *Bigint) UnmarshalYAML(value *yaml.Node) error {
	var err error
	type tmpBig Bigint

	if err := value.Decode((*tmpBig)(b)); err != nil {
		return err
	}

	b, err = FromString(value.Value)
	return err

}
```

Therer is a problem here, value.Kind expects an **yaml.ScalarNode** type, but the converted type is not. Here is the generated error:

>> cannot unmarshal !!str `1000` into bigint.tmpBig

Reply:
My guess is that it should be:

```go
var _ yaml.Unmarshaler = (*Bigint)(nil)

func (b *Bigint) UnmarshalYAML(value *yaml.Node) error {
	var str string
	if err := value.Decode(&str); err != nil {
		return err
	}

        var err error
	b, err = FromString(str)
	return err
}
```

But we could just ignore this problem if we can't solve it now... First we need to get the code in and iterate on it later.

Reply:
It soothes the panic, but the value is not being loaded, remains zero while it should be i.e. 100.